spec/jobs/send_application_submission_email_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe SendApplicationSubmissionEmailJob, type: :job do
+  let(:course) { create(:course, :leading_teaching) }
+  let(:application) { create(:application, course:, raw_application_data: { "funding_amount" => "123" }) }
+
+  subject(:job) { described_class.new(application:, email_template: "b8b53310-fa6f-4587-972a-f3f3c6e0892e") }
+
+  describe "#perform" do
+    context "when ecf_api_disabled flag is toggled off" do
+      before { Flipper.disable(Feature::ECF_API_DISABLED) }
+
+      it "returns nil" do
```

Comment:
Would it be better to test that `application_submitted_mail` isn't called here? the return value might be `nil` regardless 🤔